lib/src/strings.dart

```diff
@@ -110,6 +107,20 @@ const sizemap_native_mapping = <String, int>{
   Enum: clang.CXTypeKind.CXType_Enum
 };
 
+// Library imports.
+const libraryImports = 'library-imports';
+
+final predefinedLibraryImports = {
+  ffiImport.name: ffiImport,
+  // ffiPkgImport.name: ffiPkgImport // TODO: add this import.
```

Comment:
nit: left TODO without issue number

Reply:
This TODO will be removed once we get a dev `package:ffi`.